test/parallel/test-debugger-repeat-last.js

```diff
@@ -10,7 +10,7 @@ const fixture = path.join(
 
 const args = [
   'debug',
-  `--port=${common.PORT}`,
+  '--port=0',
```

Comment:
This might be one of those cases where the best thing to do is to move the test from `parallel` to `sequential`.